src/content/smart-bulletins/029.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "Smart Bulletin No. 029"
+intro: "Interim Federal Acquisition Regulation (FAR) Rule Prohibiting Acquiring Certain Telecommunications and Video Surveillance Equipment or Services"
+number: "No. 029"
+order: 029
+businessline: "Purchase and Integrated (when used as a purchase card)"
+published: "Aug 13, 2019"
+lastupdated: "Jun 2, 2020"
+---
+
+UPDATE:
+| Date | Version History Action Log - Summary of Action/Changes |
+| ----------- | ----------- |
+| August 13, 2019| Implementation of Smart Bulletin #29 |
+| June 2, 2020 | Updated FAQ |
+
+## Effective Date
+
+August 13, 2019
+
+## Business Line(s) Affected
+
+Purchase and Integrated (when used as a purchase card)
+
+## Introduction
+
+The Department of Defense, General Services Administration, and the National Aeronautics and Space Administration issued an interim rule amending the FAR to implement section 889(a)(1)(A) of the John S. McCain National Defense Authorization Act for Fiscal Year 2019 (Public Law 115-232).
```

Comment:
The Department of Defense, General Services Administration, and the National Aeronautics and Space Administration issued an interim rule amending the FAR to implement section 889(a)(1)(A) of the John S. McCain National Defense Authorization Act for Fiscal Year 2019 (Public Law 115-232).
Effective August 13, 2019, the interim rule revises the FAR and prohibits agencies from procuring, obtaining, extending, or renewing a contract (including micro-purchases) for equipment, systems, or services that use covered telecommunications and/or video surveillance equipment or services. This includes items that are substantial or essential to systems or are critical to any system.